### ISSUE TITLE
Failure to download wifi.apk causing the build to fail

### ISSUE THREAD

**Issue by warraagal** [linaro-swg/optee_android_manifest#56]
I am using the master branch.
There were no errors in syncing.
Here are the last few lines in the build log :

make[1]: Leaving directory '/home/rounaka/optee_android_manifest/out/target/product/hikey960/obj/kernel'
make: Leaving directory '/home/rounaka/optee_android_manifest/kernel/linaro/hisilicon-4.14'
[  0% 88/97157] build out/target/product/hikey960/hi3660-hikey960.dtb
[  0% 89/97157] build out/target/product/hikey960/dt.img
[  0% 90/97157] build out/target/product/hikey960/vendor/etc/wifi/wifi.apk
FAILED: out/target/product/hikey960/vendor/etc/wifi/wifi.apk 
/bin/bash -c "curl -s -L http://testdata.validation.linaro.org/apks/wifi/wifi.apk -o out/target/product/hikey960/vendor/etc/wifi/wifi.apk"
ninja: build stopped: subcommand failed.

**Comment by vchong** [comment]
I'm able to download it fine. Maybe be a (temporary) network issue somewhere? Please try again.

Are you using the master branch of https://github.com/linaro-swg/optee_android_manifest or AOSP? In any case, please note that we don't support the AOSP master branch anymore, only the single fixed-version quarterly release at https://github.com/linaro-swg/optee_android_manifest. Thanks.

**Comment by warraagal** [comment]
@vchong 
I am using master branch of https://github.com/linaro-swg/optee_android_manifest
I have tried many times in the last two days. It keeps failing.
I tried running the curl command separately outside of the build to find out more about what is happening.
The weird thing is that it is consistently failing at the same byte offset - 45500 every single time. I tried two different versions of curl - 7.58 and 7.47
It fails in Firefox as well, but when I change the link to https://testdata.linaro.org//apks/wifi/wifi.apk, it works. TLS certificate validation is failing in curl, so I can't use this functioning link with curl.

Using wget instead of curl worked. It did fail again at the same byte offset, but retried and continued the partial download automatically.
Retrying curl with "-C -" option works too.

I am closing this issue as I now have a temporary fix.
